### PR TITLE
Initial init changes for dynamic config.

### PR DESCRIPTION
**Pull request by tanmayja**

### CODE REVIEW

File: cmd/update-conf.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/aerospike/aerospike-kubernetes-init/pkg"
+)
+
+// confUpdate represents the update-conf command
+var confUpdate = &cobra.Command{
+	Use:   "update-conf",
+	Short: "update conf init functionality",
+	Long:  `This command updates conf file of aerospike server and updating CR status.`,
```
Comment (abhishekdwivedi3060) [inline cmd/update-conf.go:30]:
```suggestion
	Short: "update conf init functionality",
	Long:  `This command updates conf file of aerospike server and CR status.`,
```

**Review by abhishekdwivedi3060 — APPROVED** [review]

File: pkg/update_pod_status_util.go [hunk 2]
```diff
@@ -597,6 +598,32 @@ func (initp *InitParams) manageVolumesAndUpdateStatus(ctx context.Context, resta
...
+	}
+
+	pod.Annotations["aerospikeConf"] = string(data)
+
+	if err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
+		// Customize the error check for retrying, return true to retry, false to stop retrying
+		return true
+	}, func() error {
+		// Patch the resource
+		if err := initp.k8sClient.Update(ctx, pod); err != nil {
```
Comment (abhishekdwivedi3060) [inline pkg/update_pod_status_util.go:617]: I think we should get the latest object first here to fix object already modified errors also

File: pkg/update_pod_status_util.go [hunk 3]
```diff
@@ -597,6 +598,32 @@ func (initp *InitParams) manageVolumesAndUpdateStatus(ctx context.Context, resta
...
+			return err
+		}
+
+		initp.logger.Info("Pod status patched successfully", "podname", initp.podName)
+		return nil
+	}); err != nil {
+		return err
+	}
+
 	initp.logger.Info("Updating pod status", "podname", initp.podName)
```
Comment (abhishekdwivedi3060) [inline pkg/update_pod_status_util.go:627]: Fix the logs statements here. Either say `patched/updated pod successfully`  or `updated pod info in Aerospike <name> status.

**Review by abhishekdwivedi3060 — CHANGES_REQUESTED** [review]

File: pkg/update_pod_status_util.go (hunk 2 above)
Comment (abhishekdwivedi3060) [reply pkg/update_pod_status_util.go:621]: Above annotations code will also go inside retry func

File: go.sum
Comment (abhishekdwivedi3060) [inline go.sum:1]: Why is this removed

**Review by abhishekdwivedi3060 — APPROVED** [review]

**Review by abhishekdwivedi3060 — APPROVED** [review]

**Review by sud82 — CHANGES_REQUESTED** [review]

**Review by sud82 — APPROVED** [review]

**Pull request merged**